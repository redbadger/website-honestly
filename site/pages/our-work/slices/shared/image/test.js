// @flow

import React from 'react';
import { shallow } from 'enzyme';

import Image from '.';

describe('site/pages/our-work/slices/shared/image', () => {
  it('renders the correct class by default', () => {
    const component = shallow(<Image src="path/to/image" alt="" />);
    expect(component).toMatchSnapshot();
  });

  it('renders the restricted class', () => {
    const component = shallow(<Image src="path/to/image" alt="" layout="restricted" />);
    expect(component).toMatchSnapshot();
  });
});
