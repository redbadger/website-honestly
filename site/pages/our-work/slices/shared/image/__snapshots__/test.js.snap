// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`site/pages/our-work/slices/shared/image renders the correct class by default 1`] = `
<div
  className="base"
>
  <img
    alt=""
    src="path/to/image"
  />
</div>
`;
